modules/sharedaddy/recaptcha.php

```diff
@@ -128,11 +128,23 @@ public function verify( $response, $remote_ip ) {
 		if ( true !== $resp_decoded['success'] ) {
 			return new WP_Error( $error_code, $error_message );
 		}
-
 		// Validate the hostname matches expected source
 		if ( isset( $resp_decoded['hostname'] ) ) {
 			$url = wp_parse_url( get_home_url() );
-			if ( $url['host'] !== $resp_decoded['hostname'] ) {
+
+			/**
+			 * Allow other valid hostnames.
+			 *
+			 * This can be useful in cases where the token hostname is expected to be
+			 * different from the get_home_url (ex. AMP recaptcha token contains a different hostname)
+			 *
+			 * @since 9.1.0
```

Comment:
Could you add this to the docblock? 

```suggestion
			 * @module sharedaddy
			 *
			 * @since 9.1.0
```

It allows us to sort hooks by feature in the Jetpack codex.

Reply:
Sure thing!